vector/src/main/java/im/vector/fragments/VectorSettingsPreferencesFragment.java

```diff
@@ -805,6 +807,29 @@ public boolean onPreferenceChange(Preference preference, Object newValue) {
             }
         });
 
+
+        final CheckBoxPreference useRageShakeModePref = (CheckBoxPreference) findPreference(PreferencesManager.SETTINGS_USE_RAGE_SHAKE_KEY);
+        final boolean mIsUsedRageShake = PreferencesManager.useRageshake(appContext);
+
+        if(mIsUsedRageShake){
```

Comment:
The rest of code uses one space between parenthesises and brackets. Please conform to it.